### PR TITLE
Added literal-comparison message example

### PR DESCRIPTION
**Pull request by matusvalo**
Co-authored-by: Vladyslav Krylasov <vladyslav.krylasov@gmail.com>

## Type of Changes

|     | Type                   |
| --- | ---------------------- |
|     | :bug: Bug fix          |
|     | :sparkles: New feature |
|     | :hammer: Refactoring   |
| ✓   | :scroll: Docs          |

## Description

Related #5953

### CODE REVIEW

File: doc/data/messages/l/literal-comparison/bad.py [hunk 1]
```diff
@@ -0,0 +1,3 @@
+value = 25
+if value is 25:  # [literal-comparison]
+    pass
```
Comment (Pierre-Sassoulas) [inline doc/data/messages/l/literal-comparison/bad.py:3]:
```suggestion
def is_an_orange(fruit):
    return fruit is "orange" # [literal-comparison]
```

File: doc/data/messages/l/literal-comparison/good.py [hunk 2]
```diff
@@ -0,0 +1,3 @@
+value = 25
+if value == 25:
+    pass
```
Comment (Pierre-Sassoulas) [inline doc/data/messages/l/literal-comparison/good.py:3]:
```suggestion
def is_an_orange(fruit):
    return fruit == "orange"
```

**Review by Pierre-Sassoulas — CHANGES_REQUESTED** [review]
I think a function is better sometime so we have an "unknown" value and meaningful code.

**Comment by Pierre-Sassoulas** [comment]
We should probably link to : https://docs.python.org/3/library/stdtypes.html#comparisons

**Comment by matusvalo** [comment]
> We should probably link to : https://docs.python.org/3/library/stdtypes.html#comparisons

done

**Review by Pierre-Sassoulas — APPROVED** [review]

**Pull request merged**